profiler/profiler.go

```diff
@@ -192,6 +194,21 @@ func newProfiler(opts ...Option) (*profiler, error) {
 	if cfg.logStartup {
 		logStartup(cfg)
 	}
+	var tags []string
+	for _, tag := range cfg.tags.Slice() {
+		// If the user configured a tag via DD_VERSION or WithVersion,
+		// override any version tags the user provided via WithTags,
+		// since having more than one version tag breaks the comparison
+		// UI
```

Comment:
Q: It seems like we still allow the user to set multiple version tags via `WithTags`. Is that intentional?

Reply:
Ah, good point. We don't normally de-duplicate tags, but I think that'd be user-friendly to do here to match the intention of only ever having one.